iwyu_test_util.py

```diff
@@ -56,6 +56,11 @@
 _IWYU_TEST_PREREQ_RE = re.compile(r'^// IWYU_(REQUIRES|UNSUPPORTED): '
                                   r'([a-z][a-z0-9_-]*)\(([^)]+)\)$')
 
+# This is a line that specifies conditions under which a test is expected to
+# fail.
+# // IWYU_XFAIL: feature(arg)
```

Comment:
I'm thinking it might be nice to add unconditional XFAILs too to be able to add testcases for known problems, e.g.:
```
// see https://github.com/include-what-you-use/include-what-you-use/issues/1181
// IWYU_XFAIL
// IWYU_ARGS: ...

...
```

But we can follow up with that later.

Reply:
Cool, thanks!